lib/vimeo.js

```diff
@@ -456,13 +456,22 @@ Vimeo.prototype.generateClientCredentials = function (scope, fn) {
  *
  * https://developer.vimeo.com/api/reference/videos#upload_video
  *
- * @param {string}    file              Path to the file you wish to upload.
- * @param {Object=}   params            Parameters to send when creating a new video (name,
- *                                      privacy restrictions, etc.). See the API documentation for
- *                                      supported parameters.
- * @param {Function}  completeCallback  Callback to be executed when the upload completes.
- * @param {Function}  progressCallback  Callback to be executed when upload progress is updated.
- * @param {Function}  errorCallback     Callback to be executed when the upload returns an error.
+ * .upload( file [, params] [, completeCallback], progressCallback [, errorCallback])
+ *
+ * -  params (optional)
+ *    If an object is not provided, we will use the default upload params.
```

Comment:
Maybe chat with Marc on this - usually docs like this are in the passive voice (i.e. the `default upload params are used`) rather than active (i.e. `we will use the default`)